nanny.toml

```diff
@@ -35,6 +35,13 @@ to=""
 enabled=false
 webhookURL=""
 
+[webhook]
+enabled=false
+webhookURL="https://webhook.somewhere.com/webhook"
+webhookURL_all_clear="https://webhook.somewhere.com/webhook"
```

Comment:
Please keep these config names consistent `snake_case`, https://github.com/lunemec/nanny/pull/38/files#diff-ab967ab1a2f3a1b769106eeb7bfe892ef0e81d1d27811fa15be08e6749feee1fR80-R86

Reply:
I have changed the configuration names to snake_case.